### DISCUSSION TITLE
PlayStore downloads failed

### DISCUSSION THREAD

**Discussion by xtrasrx** [AdguardTeam/AdGuardHome#2751] [Q&A]
When updating the filter list, it blocks the PlayStore downloads.
I fix this by disabling the AdGuarHome service

**Comment by ghost** [comment]
Then you are blocking a domain needed by the PlayStore, I advise you to have AGH running, load the store on your PlayStation, start to download something, and then check the query logs in AGH to see what Sony or PlayStation domain is blocked and whitelist it.
